### PR TITLE
Refactor packet foractory to handle IDB

### PR DESCRIPTION
**Pull request by samaloney**

### CODE REVIEW

**Comment by samaloney** [comment]
@nicHoch what do you think, it's not perfect but a little bit cleaner

File: stixcore/tmtc/packets.py [hunk 1]
```diff
@@ -214,6 +183,7 @@ class GenericPacket:
 
     """
     _registry = dict()
+    idb_manger = None
```
Comment (nicHoch) [inline stixcore/tmtc/packets.py:186]: typo or common abbreviation?

**Review by nicHoch — APPROVED** [review]
all good

File: stixcore/tmtc/packets.py (hunk 1 above)
Comment (samaloney) [reply stixcore/tmtc/packets.py:186]: typo for sure

**Pull request merged**